core-tests/src/test/java/org/mule/runtime/core/transformer/types/DataTypeBuilderTestCase.java

```diff
@@ -309,7 +314,7 @@ public void cacheClean() throws InterruptedException, ClassNotFoundException {
     DataType.builder().type(custom.loadClass(Message.class.getName())).build();
     custom = null;
 
-    new PollingProber().check(new JUnitLambdaProbe(() -> {
+    new PollingProber(10000, 100).check(new JUnitLambdaProbe(() -> {
```

Comment:
Can constants be used here? Are there any existing contasts in abstract test cases?